tests/udf-server/udf.py

```diff
@@ -0,0 +1,57 @@
+import pyarrow as pa
+import pyarrow.flight
+
+
+def str_cmp(s1: str, s2: str):
```

Comment:
it looks simple enough.

Reply:
Yes. We only support simple types now. More types will be supported later.